### ISSUE TITLE
Unexpected error

### ISSUE THREAD

**Issue by jdyb** [rodjek/puppet-lint#942]
Hi!

Got the following and followed the instructions.
Anything more I can help with?

Best wishes,
Jon Dybeck

---
puppet-lint version: 2.4.2
ruby version: 2.7.3-p183
platform: x86_64-linux-musl
file path: manifests/init.pp
file contents:
```
# == Class: cwsens
class puppet_cwsens(
) {
  ensure_packages(['mosquitto'], { ensure => latest })
  $mosquitto_conf = @("EOT"/L)
    per_listener_settings true
    max_inflight_messages 0
    sys_interval 2
    allow_anonymous false
  | - EOI
  file { '/etc/mosquitto/mosquitto.conf':
    ensure  => present,
    owner   => root,
    group   => root,
    mode    => '0644',
    content => $mosquitto_conf,
    require => Package['mosquitto'],
  }
  service { 'mosquitto':
    ensure  => running,
    enable  => true,
    require => [
      File['/etc/mosquitto/mosquitto.conf'],
      Package['mosquitto'],
    ],
  }
}
```
error:
```
